res/database/Default/user/scr0llbaer_constr_roles.xml

```diff
@@ -0,0 +1,343 @@
+		
+		<programmerole guid="2e1e1a95-f85f-4c57-a1ee-dca164b8d4ef" first_name="Evan" last_name="Hurt" first_name_original="Ethan" last_name_original="Hunt" gender="1" comment="https://en.wikipedia.org/wiki/Ethan_Hunt" />
+		
+		<programmerole guid="463e4c3a-9299-496f-a86d-88890716b6d5" first_name="" last_name="Scook" title="Mr." first_name_original="" last_name_original="Spock" gender="1" comment="https://en.wikipedia.org/wiki/Spock" />
+		
+		<programmerole guid="716548bc-4bff-446a-b166-f58747e5fd15" first_name="Rinnetooth" last_name="" first_name_original="Winnetou" last_name_original="" gender="1" comment="https://en.wikipedia.org/wiki/Winnetou" />
+		<programmerole guid="90ef4345-6ad0-41f4-a664-4a245d22b979" first_name="Old Smasherfist" last_name="" first_name_original="Old Shatterhand" last_name_original="" gender="1" comment="https://en.wikipedia.org/wiki/Old_Shatterhand" />
+		
+		<programmerole guid="b7ce4bce-9b0c-45e0-9f27-d810d91556fa" first_name="Kelvin" last_name="McCauleyster" first_name_original="Kevin" last_name_original="McCallister" gender="1" comment="https://en.wikipedia.org/wiki/Home_Alone_(franchise)" />
+
+		<programmerole guid="e81a30f2-ce97-43a6-b55e-072fe40c712a" first_name="Jake" last_name="Pond" nick_name="0013" nick_name_original="007" first_name_original="James" last_name_original="Bond" gender="1" comment="https://en.wikipedia.org/wiki/James_Bond" />
+
+		<programmerole guid="cc9b4759-4e2c-4e6a-9241-0702d49c6df6" first_name="Bryce" last_name="Bayne" nick_name="Batguy" nick_name_original="Batman" first_name_original="Bruce" last_name_original="Wayne" gender="1" comment="https://en.wikipedia.org/wiki/Batman" />
+		<programmerole guid="ddb03dc8-2911-435e-b059-f0a4139eae0c" first_name="Doug" last_name="Greyman" nick_name="Ronny" nick_name_original="Robin" first_name_original="Dick" last_name_original="Grayson" gender="" comment="https://en.wikipedia.org/wiki/Dick_Grayson" />
+		<programmerole guid="da1380a8-c044-446b-b4b0-e1d8ceb1e9fc" first_name="Beatrice" last_name="Gardon" nick_name="Batgal" nick_name_original="Batgirl" first_name_original="Barbara" last_name_original="Gorden" gender="2" comment="https://en.wikipedia.org/wiki/Barbara_Gordon" />
+		<programmerole guid="e9e8f347-4383-471f-8426-ec1db79c60e6" first_name="Kay" last_name="Kent" nick_name="Batlady" nick_name_original="Batwoman" first_name_original="Kate" last_name_original="Kane" gender="2" comment="https://en.wikipedia.org/wiki/Batwoman" />
+		<programmerole guid="1a3faeb1-691c-4957-b472-854a36fd81dc" first_name="Serena" last_name="Keith" nick_name="Catlady" nick_name_original="Catwoman" first_name_original="Selina" last_name_original="Kyle" gender="2" comment="https://en.wikipedia.org/wiki/Catwoman" />
+
+			<!-- Cartoon characters, using readable IDs because they likely won't be referenced via actor -->
```

Comment:
What would be the problem if they were?

Reply:
Just convenience. For many cartoon programmes, actors aren't and won't be specified, hence there'll be no shortcut reference "role of actor 3".